src/flamegraph.cpp

```diff
@@ -805,13 +804,26 @@ FlameGraph::FlameGraph(QWidget* parent, Qt::WindowFlags flags)
             searchInput->setMinimumWidth(200);
             layout->addWidget(searchInput);
 
+            auto regexCheckBox = new QCheckBox(widget);
+            regexCheckBox->setText(tr("Regex Search"));
```

Comment:
can you show a screenshot please - this sounds pretty wide to me - can we somehow reduce  the width that this checkbox occupies I wonder?

Reply:
I mean I could shorten the text to something like `^[A-Z]$` and then add a tooltip explaining that this enables the regex search.

Reply:
Even for a programmer that looks strange.
That said - the version from vscode does look quite nice (inline toggle - could still be an outside checkbox) with three short "text symbols" and the expected tooltip:

![vscode search](https://github.com/user-attachments/assets/7f38e273-f349-4fbd-9134-51cfe8f11fcb)

Regex Text Symbol `.*`

Reply:
lgtm, thanks